### PR TITLE
0.36 Update

### PR DESCRIPTION
**Pull request by CalhounDog**
Update to Containers, Bloatfly, And several small bug fixes.

### CODE REVIEW

**Comment by CalhounDog** [comment]
0.36 Update ship to download page

**Comment by milesflo** [comment]
Can you make each of those changes their own string?

File: src/views/Download.tsx [hunk 1]
```diff
@@ -12,6 +12,14 @@ interface IBuildInfo {
 
 class Download extends React.Component {
   public buildsList: IBuildInfo[] = [
+     {
+      changelog: [
+        "Fixed Bloatfly Damage, Looting Menu/Containers Fix, Third Person transition bug fix, Fixed Aiming Desync ",
```
Comment (milesflo) [inline src/views/Download.tsx]: Can you make these their own strings? 

ex:
```js
[
    "Fixed Bloatfly Damage",
    "Looting Menu/Containers Fix",
    "Third Person transition bug fix"
    // etc
```

That way the bullet points on the website look good

Comment (CalhounDog) [reply src/views/Download.tsx]: Yeah much better. How do I modify the commit?

Comment (milesflo) [reply src/views/Download.tsx]: Head [here](https://github.com/atomicmp/atomicmp-site/pull/13/files) and click on the 3 dots to the right of the Downloads.tsx header... There'll be an edit option and I believe it'll get added on to this PR.

**Comment by CalhounDog** [comment]
Did that work?

**Pull request merged**